src/components/policy/policy_external/include/policy/policy_table/enums.h

```diff
@@ -189,408 +192,6 @@ const char* EnumToJsonString(HybridAppPreference val);
 bool EnumFromJsonString(const std::string& literal,
                         HybridAppPreference* result);
 
-/**
- * @brief Enumeration FunctionID.
- *
- * Enumeration linking function names with function IDs in AppLink protocol.
- * Assumes enumeration starts at value 0.
- */
-enum FunctionID {
```

Comment:
Was this unused?

Reply:
actually, there was a slight enhancement introduced in scope of this feature. some policy types are now dynamically generated from API types. hardcoded values were removed

Reply:
@mked-luxoft Where does this happen?

Reply:
@JackLivio generation happens during build with python script, similarly as API types